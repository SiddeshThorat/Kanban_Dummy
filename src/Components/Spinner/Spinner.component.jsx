import React from 'react';
import './Spinner.styles.css';

const Spinner = () => (
    <div className="spinnerOverlay">
       <div className="spinnerContainer" />
    </div>
);

export default Spinner;