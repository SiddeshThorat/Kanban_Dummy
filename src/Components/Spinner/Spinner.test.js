import React from 'react';
import { shallow } from 'enzyme'
import Spinner from'./Spinner.component';

describe("Testing Spinner component",() => {
    it("Snapshot testing of Spinner component",() => {
        expect(shallow(<Spinner />)).toMatchSnapshot();
    })
})


