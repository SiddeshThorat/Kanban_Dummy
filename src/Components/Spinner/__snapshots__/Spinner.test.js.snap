// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Spinner component Snapshot testing of Spinner component 1`] = `
<div
  className="spinnerOverlay"
>
  <div
    className="spinnerContainer"
  />
</div>
`;
